src/ResourceManager/KeyVault/Commands.KeyVault/Models/ModelExtensions.cs

```diff
@@ -117,6 +117,11 @@ public static string GetDisplayNameForADObject(string objectId, ActiveDirectoryC
                         displayName = servicePrincipal.DisplayName;
                         upnOrSpn = servicePrincipal.ServicePrincipalNames.FirstOrDefault();
                     }
+                    else if (obj.Type.Equals("group", StringComparison.InvariantCultureIgnoreCase))
```

Comment:
@maddieclayton we should add a change log entry if we want this fix to go out to users before Build